sdk/appcenter-distribute/src/debug/res/values/features.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="install_id_feature_enabled">True</string>
```

Comment:
Add `appcenter_distribute_` to every string resource as this gets injected into customer resources. We need proper namespacing. We do that also for texts.